setup-buildengine.sh

```diff
@@ -271,13 +267,11 @@ checkForRequiredFiles() {
         fatal "Tooling file [$OPT_TOOLING] not found in the current directory."
     fi
 
-    if [[ ! -f $OPT_TARGET_ARM ]]; then
-        fatal "Target file [$OPT_TARGET_ARM] not found in the current directory."
-    fi
-
-    if [[ ! -f $OPT_TARGET_I486 ]]; then
-        fatal "Target file [$OPT_TARGET_I486] not found in the current directory."
-    fi
+    for targetarch in ${!OPT_TARGET_FILENAMES[*]} ; do
+        if [ ! -e "${OPT_TARGET_FILENAMES[$targetarch]}" ] ; then
```

Comment:
We are already looping through the filenames

Reply:
What I mean is that it would make more sense to simply:
```
for target_filename in ${OPT_TARGET_FILENAMES[*]} ; do
    if [ ! -e $target_filename ]; ... 
```